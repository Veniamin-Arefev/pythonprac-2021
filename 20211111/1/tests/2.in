A, B, C = OrderedComplex(-14, 125), OrderedComplexMul(832, -114),  OrderedComplexMul(-1, -2)
print(A < B, B @ B, OrderedComplex(C @ C, B @ B) < A)
