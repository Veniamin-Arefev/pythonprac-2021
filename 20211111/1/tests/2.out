False 705220.0 False
